src/v/redpanda/admin_server.cc

```diff
@@ -1091,8 +1091,8 @@ void admin_server::register_broker_routes() {
                           return nhr.id == id;
                       });
                     if (r_it != h_report.value().node_reports.end()) {
-                        it->second.version = r_it->redpanda_version;
-                        for (auto& ds : r_it->disk_space) {
+                        it->second.version = r_it->local_state.redpanda_version;
+                        for (auto& ds : r_it->local_state.disk_space) {
```

Comment:
this change looks like it needs to be in the previous commit which does the rename. otherwise, the previous commit won't compile.

Reply:
Good catch. Would be nice if CI would catch this.